website/content/docs/datasources/hcp/hcp-packer-version.mdx

```diff
@@ -0,0 +1,96 @@
+---
+
+<BadgesHeader>
+  <PluginBadge type="official" />
+  <PluginBadge type="hcp_packer_ready" />
+</BadgesHeader>
+
+# HCP Packer Version Data Source
+
+Type: `hcp-packer-version`
+
+The `HCP Packer Version` Data Source retrieves information about
+HCP Packer Version from the HCP Packer Registry. This information can be used to
+query HCP for a source external identifier for various Packer builders.
+
+To get started with HCP Packer, refer to the [HCP Packer documentation](/hcp/docs/packer) or try the
+[Get Started with HCP Packer tutorials](/packer/tutorials/hcp-get-started).
+
+~> **Note:** You will receive an error if you try to reference metadata from a deactivated or deleted registry.
+An administrator can manually deactivate or delete a registry, and HCP Packer automatically deactivates registries
+with billing issues. Contact [HashiCorp Support](https://support.hashicorp.com/) with questions.
+
+## Revoked Versions
+
+If an HCP Packer Version is revoked, the `hcp-packer-version` data source will fail and Packer won't proceed with
+the build. Building new artifacts from a revoked artifact is not compliant.
+Versions that are scheduled to be revoked will still be considered valid until the revocation date.
+
+## Basic Example
+
+Below is a fully functioning example. It stores information about an HCP Packer Version, which can then be accessed as a variable.
+
+```hcl
+data "hcp-packer-version" "hardened-source" {
+  bucket_name  = "hardened-ubuntu-16-04"
+  channel_name = "dev"
+}
+```
+
+## Full Example
+
+This data source can be used in conjunction with the `hcp-packer-artifact`
+data source to retrieve an artifact identifier. You provide the version fingerprint and channel
+name to the version data source, then use the version source inside the
+artifact data source, then use the artifact data source inside your source block.
+
+```hcl
+# Retrieves information about the HCP Packer Version; a "version" can be
+# thought of as all the metadata created by a single call of `packer build`.
+data "hcp-packer-version" "hardened-source" {
+  bucket_name  = "hardened-ubuntu-16-04"
+  channel_name = "dev"
+}
+
+# Retrieves information about the HCP Packer Artifact; an artifact can be thought
+# of as all the metadata (including the artifact names) created by a single
+# "source" builder; this can include multiple artifacts so we provide a
+# region to disambiguate.
+data "hcp-packer-artifact" "foo" {
+  bucket_name         = "hardened-ubuntu-16-04"
+  version_fingerprint = data.hcp_packer_version.hardened-source.fingerprint
+  platform            = "aws"
+  region              = "us-east-1"
+}
+
+# This source uses the output from a previous Packer build. By using the
+# HCP Packer Registry in this way, you can easily create build pipelines where
+# a single base artifact can be customized in multiple secondary layers.
+source "amazon-ebs" "packer-secondary" {
+  source_ami       = data.hcp-packer-artifact.foo.id
```

Comment:
Lets try not to use foo in the example. Use either example or a name that people would use in the template to describe the infrastructure as code. 
```suggestion
  source_ami       = data.hcp-packer-artifact.example.id
```